src/typeahead/typeahead.ts

```diff
@@ -89,6 +89,12 @@ export class NgbTypeahead implements ControlValueAccessor,
   private _windowRef: ComponentRef<NgbTypeaheadWindow>;
   private _zoneSubscription: any;
 
+  /**
+   * Value for the configurable autocomplete attribute.
+   * Defaults to 'off' to disable the native browser autocomplete, but this standard value does not seem
+   * to be always correctly taken into account (cf https://github.com/ng-bootstrap/ng-bootstrap/issues/2194).
```

Comment:
We are generating user documentation from those JS doc comments and I don't think that we want to have links to GH issues in the users docs.

Could you please split this message into 2 parts:
- relevant for end-users and keep it here;
- relevant for project developers and move it to the commit message?